lib/main.dart

```diff
@@ -17,10 +19,11 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return GetMaterialApp(
+      initialRoute: HomePageRoute,
       debugShowCheckedModeBanner: false,
       title: "SoundScape",
       theme: ThemeData(
-          scaffoldBackgroundColor: Colors.white,
+          scaffoldBackgroundColor: light,
```

Comment:
cool progress on what I assume will eventually be the light dark mode, I can implement the dark though. You work to hard